test/per_file_coverage.sh

```diff
@@ -59,6 +59,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/wasm_runtime/wasmtime:0.0" # Not enabled in coverage build
 "source/extensions/wasm_runtime/wavm:0.0" # Not enabled in coverage build
 "source/extensions/watchdog:83.3" # Death tests within extensions
+"source/extensions/listener_managers:70.5"
```

Comment:
Wow, this is quite low. It looks like the only change in that directory is a path in a #include. Do you understand why the test coverage is so low?

Reply:
this directory covered the highly tested TCP code and the not highly tested validation manager code.